smoke_tests/standalone_smoke_test.py

```diff
@@ -0,0 +1,324 @@
+# Copyright 2014-2020 Scalyr Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.K
+
+from __future__ import absolute_import
+from __future__ import unicode_literals
+from __future__ import print_function
+
+import datetime
+import time
+import json
+import re
+
+import pytest  # type: ignore
+
+from scalyr_agent.__scalyr__ import PACKAGE_INSTALL, DEV_INSTALL
+from scalyr_agent import compat
+
+from smoke_tests.tools.agent_runner import AgentRunner
+from smoke_tests.tools.request import ScalyrRequest
+
+from six.moves import range
+import six
+
+_TIMESTAMP_PATTERN = r"\d{4}-\d{2}-\d{2}\s+\d{2}:\d{2}:\d{2}.\d+Z"
+_LEVEL_PATTERN = r"INFO|WARNING|ERROR|DEBUG"
```

Comment:
Do we enable debug log level for agent during those runs?

It appears we don't, right? If we do, we should also introspect / follow agent_debug.log log file for messages.

Reply:
Yes, we don't, but It is a good idea to enable it. But I think this should be added in a separate PR. Debug log can contain binary length prefixed strings and to read those we need to return some of the `jsonlib` features.